src/pages/Subscription/components/SubCard.js

```diff
@@ -3,10 +3,30 @@ import styled from 'styled-components';
 import Card from '../../../components/Card';
 import Button from '../../../components/Button';
 import { useNavigate } from 'react-router-dom';
+import { API } from '../../../config';
 
 const SubCard = ({ card }) => {
   const navigate = useNavigate();
 
+  const autoKakaPay = (id) => {
+    fetch(`${API.SUBSCRIPTION}`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization:
+          'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOjEsImlhdCI6MTY3NTU4NDcyNX0.tylNeMdSFT6iAxe6kJyfuoDpKduEoik4kTYM_PfiW1c',
+        'Access-Control-Allow-Origin': '*',
+      },
+      body: JSON.stringify({
+        subscriptionTypeId: id,
+      }),
+    })
```

Comment:
map을 통해서 id를 각각 뿌리는 게 아니라 컴포넌트 자체가 하나의 card를 갖고 있다면,
굳이 인자로 받지 않고도 card.id에 직접 접근할 수도 있겠네요.

```suggestion
  const autoKakaPay = () => {
    fetch(`${API.SUBSCRIPTION}`, {
      method: 'POST',
      headers: {
        'Content-Type': 'application/json',
        Authorization: localstorage.getItem('accessToken'),
        'Access-Control-Allow-Origin': '*',
      },
      body: JSON.stringify({
        subscriptionTypeId: card.id,
      }),
    })
```

Reply:
accessToken도 항상 주의!